### ISSUE TITLE
[UI005][UI] As a User, I should be able to view all the data scraped from Google

### ISSUE THREAD

**Issue by AetheryNA** [AetheryNA/google_scraper_ruby#9]
## Why

The user should be able to view all the data scraped from google and stored under their profile. This way the user can see the data they wanted from the uploaded CSV.

## Acceptance Criteria

- Should contain a table that holds all the data for the user to view

**Comment by carryall** [comment]
@AetheryNA Do we need another issue to work on the backend part for the scraped data list? e.g. adding the controller

**Comment by AetheryNA** [comment]
Agreed, added it [here](https://github.com/AetheryNA/google_scraper_ruby/issues/15) 😸
